src/main/java/ai/elimu/dao/jpa/LetterDaoJpa.java

```diff
@@ -41,4 +41,13 @@ public List<Letter> readAllOrderedByUsage() throws DataAccessException {
             "ORDER BY l.usageCount DESC, l.text")
             .getResultList();
     }
+
+    @Override
+    public List<Letter> readAllOrderedById() throws DataAccessException {
+        return em.createQuery(
+                "SELECT l " +
+                    "FROM Letter l " +
+                    "ORDER BY l.id DESC, l.text")
```

Comment:
@AshishBagdane Can we make it _ascending_ instead of descending? That way, new content will be appended to the CSVs instead when we export them (instead of being added as a new row at the top).